google/cloud/bigquery/big_query_read_connection.gcpcxx.pb.h

```diff
@@ -78,11 +69,10 @@ class BigQueryReadConnection {
 };
 
 std::shared_ptr<BigQueryReadConnection> MakeBigQueryReadConnection(
-    BigQueryReadConnectionOptions const& options =
-        BigQueryReadConnectionOptions());
+    internal::Options const& options = internal::Options());
```

Comment:
This is the only overload that we want. Per our discussion at the team meeting, we want to avoid the other overloads that take separate policy arguments.